src/container/HashContainer/Base/index.ts

```diff
@@ -1,4 +1,4 @@
-import { Base, Container } from '@/container/ContainerBase/index';
```

Comment:
tsTreeShaker didn't handle skipping the index in the file path, so I think we should check if this code is valid for PR #35.

Reply:
I didn't understand. What should I check?

Reply:
This is a problem of #35
It seems better to merge before #35 to apply this PR because it is necessary to make it possible to import the index omitted.

In other words, this kind of import can cause a bug in #35, so I have to check.

Reply:
Performance test is doing in production environment.

It works fine wo I think this is not a problem?

Reply:
There's no problem with this